runner/src/main.rs

```diff
@@ -190,6 +190,16 @@ fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Step {
                     "--release",
                     "--target=wasm32-unknown-unknown",
                     &format!("--manifest-path={}", cargo_manifest),
+                    // Use a fixed target directory, because `--target-dir` influences SHA256 hash
+                    // of Wasm module. This directory should also be synchronized with
+                    // `--target-dir` used in [`oak_tests::compile_rust_wasm`] in order to have
+                    // same SHA256 hashes.
+                    &format!("--target-dir={}", {
+                        let mut target_dir = PathBuf::from(cargo_manifest);
+                        target_dir.pop();
+                        target_dir.push("target");
+                        target_dir.to_str().expect("Invalid target dir").to_string()
```

Comment:
Isn't this the same as not specifying any target dir?

Reply:
Without specifying the `--target-dir` it would be `examples/target` and not `examples/{}/module/rust/target`

Reply:
But wouldn't it be the same for both runner and tests, at least?

Reply:
If we will use `examples/target` then `integration_test.rs` also will be occupy the same directory with Wasm modules.
It leads to errors because we are running `cargo build` inside `cargo test`, and they are using the same target directory.

Reply:
Got it, that seems the most important thing to include in the comment then, isn't it? (sorry if I missed it)

Reply:
Done